.github/workflows/static-checks.yaml

```diff
@@ -19,13 +19,17 @@ jobs:
           - "make test"
           - "sudo -E PATH=\"$PATH\" make test"
     env:
+      # Still used in kata-containers/tests:.ci/setup.sh to select minimal
+      # package set for install
```

Comment:
I **think** this can be removed, but I need to take a look at the jenkins jobs definitions to be 100% sure.
Poke me again about this if I don't get back to you in a day or so.

Reply:
As far I `grep` could help me, we pretty much don't use the `TRAVIS` variable anywhere.
I'd say we can go ahead and remove it, and also cleanup the tests and ci repos, whenever it's mentioned.

Reply:
ok, i've opened https://github.com/kata-containers/tests/pull/5518 & https://github.com/kata-containers/ci/pull/533, and will update this commit to also remove the usage here.